app/models/operatingsystems/windows.rb

```diff
@@ -36,4 +40,9 @@ def boot_files_uri(medium_provider)
   def display_family
     "Windows"
   end
+
+  def encrypt_password(root_pass)
```

Comment:
Since this is an implementation detail in the specific template, i would rather exclude it from here and only fix the "official" api to work (and cp to 1.20-stable)